client/product_details/ProductDetails.jsx

```diff
@@ -68,10 +68,15 @@ class ProductDetails extends React.Component {
     var availableSizes = [];
     if (this.props.selectedStyle) {
       var skus = this.props.selectedStyle.skus;
+      console.log('skus: ', skus);
```

Comment:
No biggie but could remove these console.logs in a future PR!

Reply:
I'm so sorry! I completely forgot about the console statements. I will remove these.